cli/src/commands/workspace/add.rs

```diff
@@ -31,9 +32,33 @@ use crate::command_error::user_error;
 use crate::command_error::CommandError;
 use crate::ui::Ui;
 
+/// How to handle sparse patterns when creating a new workspace.
+#[derive(clap::ValueEnum, Clone, Debug, Eq, PartialEq)]
+enum SparseInheritance {
+    /// Copy sparse patterns from the current workspace.
+    Copy,
+    /// Use the default sparse pattern for a "full" repository, including all
+    /// files in the workspace.
```

Comment:
nit: maybe s/repository/working copy/

Also, it might be better to say simply say "Include all files in the working copy". That way it's still true even if we change the default.

Reply:
I reworded to make it all a little simpler. Thanks!